### DISCUSSION TITLE
State Events - Start and Stop

### DISCUSSION THREAD

**Discussion by jbenne64** [olivierfriard/BORIS#743] [General]
Hi, 
I have made the mistake of only recording the 'start' of state events in a live view. This means all my observations are paired but incorrect. Is there a way to manually make all the observations I have entered as 'start' events and manually enter the 'stop' event which is based on the 'start' of the next behaviour. Thank-you

**Comment by olivierfriard** [comment]
Hi,
you can open the live observation in view mode (Observations > View observation) and add new events (Observations > Add event or using CTRL + A)
